benchmarks/build.gradle

```diff
@@ -33,13 +33,13 @@ archivesBaseName = 'elasticsearch-benchmarks'
 test.enabled = false
 
 dependencies {
-    compile("org.elasticsearch:elasticsearch:${version}") {
+    bundle("org.elasticsearch:elasticsearch:${version}") {
```

Comment:
Why would we need to bundle anything in benchmarks?

Reply:
I have no clue. It has been shadowing all of its dependencies for ages and I was keeping it going. I could dig into it though.

Reply:
Ok, I didn't realize that. Quite odd, maybe @danielmitterdorfer can elaborate on why we do this?

Reply:
This is an area where we can improve a lot: commenting the non-obvious choices in our code.

Reply:
I think it's for convenience, so that you can run the benchmarks as simply as ` java -jar benchmarks.jar`.

Reply:
I've done a little digging and I also think it is for convenience. Every example I see of using JMH shades the dependencies but [this](http://mail.openjdk.java.net/pipermail/jmh-dev/2014-June/001031.html) says that it is not required.